static/src/javascripts/projects/commercial/modules/prebid/types.js

```diff
@@ -75,8 +76,10 @@ export type PrebidSlot = {
         | 'inline'
         | 'mostpop'
         | 'comments'
-        | 'mobile-sticky',
+        | 'mobile-sticky'
+        | 'banner',
     sizes: PrebidSize[],
+    matchesDfpSlot?: Advert => boolean,
```

Comment:
As before, this mixes behaviour and data, which in my experience means this is much more likely to be a source of frustration in the future. Is it possible to have a think about what we're trying to achieve here, and better encapsulate the behaviour? This approach forces the logic to be scattered around the codebase.

Reply:
Yes, not ideal. See comments about the same feature above :)